Terran-One/cw-simulate-ui
# Minor styling update

## Issue link

[WL-XXX](https://terran-one.atlassian.net/browse/WL-XXX)

## Description

Show state renderer full screen

## Test steps

1. Upload contract
2. Verify state renderer displays full screen.
3.
